charts/nri-kube-events/values.yaml

```diff
@@ -34,22 +34,13 @@ images:
   # - name: regsecret
 
 # -- Resources available for this pod
-resources: {}
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
-
-forwarder:
-  resources:
-    limits:
-      cpu: 300m
-      memory: 500Mi
-    requests:
-      cpu: 100m
-      memory: 150Mi
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
changing the default might have different implications. My advice would be to add the new option and keep it commented out. 

In this way the behavior would be exactly the same and there would be no issues with the upgrade test

Reply:
Thank you! I will revert this change in next commit